contributing/release-schedule.md

```diff
@@ -1,84 +1,84 @@
+[_Type: Release_ GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) are used to track the status of current and past releases (from the initial cut to **experimantal**/**beta** testing to **stable**). Announcements about releases are made on the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
 
-- In general only fixes for [P0 issues](https://github.com/ampproject/amphtml/blob/master/contributing/issue-priorities.md) (causing "an outage or a critical production issue") may be cherry picked. P0 issues are those that:
-  - cause privacy or security issues
-  - cause user data loss
-  - break existing AMP web pages in a significant way
-  - or would otherwise cause a significant harm to AMP's reputation if left unresolved
-- Regressions found in the canary that are not P0 _may_ be approved if they can be resolved with a rollback. Fixes other than rollbacks--no matter how simple they may seem--will not be approved because these can cause cascading problems and delay the release of canary to production for everyone.
+You can determine what changes are in a given build using one of the following:
 
-### Process for requesting a cherry pick
+- The [_Type: Release_ GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) for each release build will include a link to the specific [release page](https://github.com/ampproject/amphtml/releases) listing the PRs in that release.
+  - The most recent _Type: Release_ issue with _(Production)_ in the title tracks the build currently in the **stable** release.
+  - The _Type: Release_ issue with _(Canary)_ in the title tracks the build currently in the **experimental**/**beta** releases.
+- The [_PR Use: In Canary_](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Canary%22), [_PR Use: In Production_](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Production%22), and [_PR Use: In LTS_](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20LTS%22) labels are added to PRs when they've made it into a _weekly_ or **lts** build. There may be a delay between when the build is created and when the label is added.
 
-Use the following process to request a cherry pick if you have a change that you believe meets the [cherry pick criteria](#cherry-pick-criteria).
+> Note: These labels still use legacy release names. They will be renamed soon to reflect the new release channel names (**beta** and **stable**).
 
-If you run into any issues or have any questions when requesting a cherry pick, please use the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
+## Release Cadence
```

Comment:
The structure of this section is a little weird to me. Can we rearrange it to state what the cadence is and then explain why?

Reply:
I too have some changes to suggest to the overall verbiage of this section, but happy to hold off until @danielrozenberg's PR that makes these changes.

Reply:
Given the major changes you've made to these docs I guess it makes sense for you to work on this section too. I still plan on making the visualizations to help parse these dry descriptions, but these dry descriptions are important nonetheless. @rsimha if you have specific suggestions go ahead and place them here

Reply:
I'm sure I'll have several comments on this section :)

Since it is pre-existing text in this, I think it's okay to leave it as is for now and reword it in a separate PR. SG?

Reply:
Resolving this for now; when Daniel comes through with Nightly stuff, we can do a second round of structural cleanup